rllib/algorithms/impala/impala.py

```diff
@@ -371,6 +371,27 @@ def validate(self) -> None:
         # Check `entropy_coeff` for correctness.
         if self.entropy_coeff < 0.0:
             raise ValueError("`entropy_coeff` must be >= 0.0!")
+        # Entropy coeff schedule checking.
```

Comment:
Same checking for entropy coeff schedules. Scheduling is now handled centrally in the ImpalaLearner base class.

Reply:
done, added a new, framework agnostic Scheduler API that handles validation, single-fixed values, and (piecewise) schedules. Used now for lr- and entropy schedules (possibly more soon as we add more algos to the new learner API).